Gems/Multiplayer/Code/Source/MultiplayerSystemComponent.cpp

```diff
@@ -273,10 +289,22 @@ namespace Multiplayer
                 console->PerformCommand("connect");
             }
         }
+
+        m_metricsEvent.Enqueue(AZ::TimeMs{ 1000 }, true);
```

Comment:
This should probably be wrapped in a CVAR so it can be enabled/disabled. A const for the event interval wouldn't hurt either.

Reply:
Done.